packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -249,6 +273,12 @@ function resetNotebook() {
 	emptyImages();
 	updateCodeState();
 }
+function onUpdateGoalQuery(goal: string) {
+	const state = cloneDeep(props.node.state);
+	goalQuery.value = goal;
+	state.goal = goalQuery.value;
```

Comment:
It will be easier to understand what is happening.
```suggestion
	state.goal = goal;
```